CWE.Data/Models/Mute.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CWE.Data.Models
+{
+    /// <summary>
+    /// Represents a mute structure to keep track of muted users.
+    /// </summary>
+    [Keyless]
+    public class Mute
+    {
+        /// <summary>
+        /// Gets or sets the infractio id for this mute.
+        /// </summary>
+        public Guid InfractionId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the user who is muted.
+        /// </summary>
+        public ulong User { get; set; }
+
+        /// <summary>
+        /// Gets or sets the time the mute started.
+        /// </summary>
+        public DateTime MuteStart { get; set; }
+
+        /// <summary>
+        /// Gets or sets the time the mute ends.
+        /// </summary>
+        public DateTime MuteEnd { get; set; }
```

Comment:
```suggestion
        /// <summary>
        /// Gets or sets the time the mute started.
        /// </summary>
        public DateTime Start { get; set; }

        /// <summary>
        /// Gets or sets the time the mute ends.
        /// </summary>
        public DateTime End { get; set; }
```
Renaming of the properties as the class name is already mute, no need to prefix the properties with the name of the class they're in.